corehq/apps/translations/app_translations.py

```diff
@@ -783,14 +796,34 @@ def _looks_like_markdown(str):
     def get_markdown_node(text_node_):
         return text_node_.find("./{f}value[@form='markdown']")
 
+    def get_all_value_nodes(text_node_):
+        empty_form_attr_value_nodes = []
+        default_form_attr_value_nodes = []
+        for n in text_node_.findall("./{f}value"):
+            if 'form' not in n.attrib:
+                empty_form_attr_value_nodes.append(n)
+            elif n.get('form') == 'default':
+                # invalid value for form attrib seen in forms
+                default_form_attr_value_nodes.append(n)
+        assert len(empty_form_attr_value_nodes) in [0, 1]
+        assert len(default_form_attr_value_nodes) in [0, 1]
```

Comment:
What does the user see if one of these assertions fails?

Reply:
My understanding says that this should never ever fail. This is just a sanity check to ensure the assumption that there would always be just one node for both of these.